micrometer-api/src/main/java/io/micrometer/api/instrument/observation/Observation.java

```diff
@@ -227,9 +226,8 @@ default void scoped(Runnable action) {
         /**
          * Current observation available within this scope.
          *
-         * @return current observation or {@code null} if one is not present
+         * @return current observation that this scope was created by
          */
-        @Nullable
```

Comment:
Since the `Scope` can only be created by an `Observation`, it should not be possible that this is null.